File: FRONT/frontend/proyecto-Gestion-Ambiental/src/app/models/Usuario.ts
//autor:Leandro Palacios
export interface Usuario {
    id_user: number;
    nombre: string;
    apellido: string;
    correo: string;
    contrasena: string;
    rol: 'Administrador' | 'Coordinador' | 'Voluntario';
    estado: string;
}

File: BACK/backend/Proyecto-Gestion-Ambiental/databasefirst/Models/Usuario.cs
using System;
using System.Collections.Generic;

namespace databasefirst.Models;

public partial class Usuario
{
    public int id_user { get; set; }

    public string nombre { get; set; } = null!;

    public string apellido { get; set; } = null!;

    public string correo { get; set; } = null!;

    public string contrasena { get; set; } = null!;

    public string rol { get; set; } = null!;

    public string? estado { get; set; }

    public virtual ICollection<Participacion> Participacions { get; set; } = new List<Participacion>();
}
